file/shared.room.web.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "id": "ac2b235a-b1e2-4848-a687-6669a3f7551b",
+    "name": "The Dock",
+    "disableMeeting": true
+  },
+  {
+    "id": "room-2",
+    "name": "Nebuchadnezzar"
+  },
+  {
+    "id": "room-3",
+    "name": "Logos"
+  },
+  {
+    "id": "room-4",
+    "name": "Osiris"
+  },
+  {
+    "id": "room-5",
+    "name": "Icarus"
+  },
+  {
+    "id": "room-6",
+    "name": "Caduceus"
+  },
+  {
+    "id": "room-7",
+    "name": "Brahma"
+  },
+  {
+    "id": "room-8",
+    "name": "Novalis"
+  },
+  {
+    "id": "room-9",
+    "name": "Vigilant"
+  },
+  {
+    "id": "room-10",
+    "name": "Zion"
+  }
+]
```

Comment:
Whats the use proposal for this file?

Reply:
I'm putting this information in this [PR](https://github.com/ResultadosDigitais/matrix/pull/149/files#diff-550856cf67eb7260aaeb883550043af8), maybe we can bring this information for the @diegos PR

Reply:
Great idea, we can remove this sample now.

Reply:
I'll open a PR for your branch...

Reply:
Done: https://github.com/ResultadosDigitais/matrix/pull/152 can you merge it in this branch?